org.musicbrainz.Picard.json

```diff
@@ -1,56 +1,152 @@
-                    "sha256": "7eaf7a2ea7d4d38a56dd6d2506574464bddf7cf284c960801679942377c297bc"
+                    "url": "http://download.qt.io/archive/qt/5.10/5.10.1/submodules/qtbase-everywhere-src-5.10.1.tar.xz",
+                    "sha256": "d8660e189caa5da5142d5894d328b61a4d3ee9750b76d61ad74e4eee8765a969"
                 },
                 {
-                    "type": "file",
-                    "path": "sip-configure",
+                    "type": "shell",
+                    "commands": [
+                        "mv configure configure.qt"
+                    ]
+                },
+                {
+                    "type": "patch",
+                    "path": "patch/qtbase-flatpak-portals-filechooser.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "patch/qtbase-flatpak-portals-services.patch"
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "processed=`sed -e 's/--/-/g ; s/=/ /g' <<< $@`",
+                        "./configure.qt $processed"
+                    ],
                     "dest-filename": "configure"
                 }
-            ],
-            "cleanup": [
-                "/bin",
-                "/include"
             ]
         },
         {
             "name": "pyqt",
+            "config-opts": [
+                "--disable-static",
+                "--enable-x11"
+            ],
+            "no-autogen": true,
             "sources": [
                 {
                     "type": "archive",
                     /* Sourceforge URLs fail intermitently, get the tarballs from Fedora, check the upstream hashes */
-                    "url": "https://src.fedoraproject.org/repo/pkgs/rpms/PyQt4/PyQt4_gpl_x11-4.12.1.tar.gz/sha512/0651816ed5a371c24827e182eac5b2a2afe67928d95a09f63bd8158117aacd3e835b1efa06d4fc0c147a28038613a5bc088bd736a1348de68199da7952299af1/PyQt4_gpl_x11-4.12.1.tar.gz",
-                    "sha256": "45d724461ae58b019b9be95076ca07e212eb73084aec9125bb254825c3a05383"
+                    "url": "https://src.fedoraproject.org/repo/pkgs/rpms/python-qt5/PyQt5_gpl-5.10.1.tar.gz/sha512/04cd0292f7a088f9c0d088d7c73ec96cfc40253f771dfdf194eb92be7635e6fcaf04863fa24215e9c34a096658549bb2feeb8d8fdca1e39ed87f0256ef0f790b/PyQt5_gpl-5.10.1.tar.gz",
+                    "sha256": "9932e971e825ece4ea08f84ad95017837fa8f3f29c6b0496985fa1093661e9ef"
                 },
                 {
                     "type": "file",
                     "path": "pyqt-configure",
                     "dest-filename": "configure"
                 }
             ],
+            "modules": [
+                {
+                    "name": "sip",
```

Comment:
Why did you move sip after PyQt?

If there isn't a strong reason to have it after, I'd rather you didn't, to keep the diff minimal and easier to review. (and if there is a good reason for it, then please move it in its own commit)